android/manifest

```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 6.0.0
+version: 6.0.1
```

Comment:
we'll need to bump this to the major version

Reply:
there's no new functionality in the module itself and there aren't any api changes

Reply:
you're right, nm